### PR TITLE
Fix issues in BotWorker and BotWorkerConfiguration files

### PR DESCRIPTION
**Pull request by Maihav**
Fix warnings on BotWorker .cs and BotWorkerConfiguration .cs files.

### **Changes made**

Add documentation for: 
- classes
- methods
- properties

### CODE REVIEW

File: Botkit/Microsoft.BotKit/BotWorker.cs [hunk 1]
```diff
@@ -1,96 +1,107 @@
...
-using Microsoft.Bot.Schema;
-using Microsoft.BotKit.Core;
-using System;
-using System.Collections.Generic;
-using System.Dynamic;
-using System.Threading.Tasks;
-
 namespace Microsoft.BotKit
 {
+    using System;
```
Comment (ceciliaavila) [inline Botkit/Microsoft.BotKit/BotWorker.cs]: The using statements must be outside the namespace scope. 
I think this rule should be ignored.

File: Botkit/Microsoft.BotKit/BotWorker.cs [hunk 2]
```diff
@@ -1,96 +1,107 @@
...
+        /// <summary>
         /// Send a message using whatever context the bot was spawned in or set using changeContext() --
-        /// or more likely, one of the platform-specific helpers like startPrivateConversation() (Slack), startConversationWithUser() (Twilio SMS), and 
+        /// or more likely, one of the platform-specific helpers like startPrivateConversation() (Slack), startConversationWithUser() (Twilio SMS), and
         /// startConversationWithUser() (Facebook Messenger) Be sure to check the platform documentation for others - most adapters include at least one.
         /// </summary>
-        /// <param name="message">A BotkitSlackMessage containing the text of a reply, or more fully formed message object</param>
-        /// <returns>Return value will contain the results of the send action, typically {id: <id of message>}</returns>
+        /// <param name="message">A BotkitSlackMessage containing the text of a reply, or more fully formed message object.</param>
+        /// <returns>A <see cref="Task{TResult}"/>Return value will contain the results of the send action, typically {id:. <id of message>}.</returns>
```
Comment (ceciliaavila) [inline Botkit/Microsoft.BotKit/BotWorker.cs]: Is there an extra '.' here?

File: Botkit/Microsoft.BotKit/BotWorkerConfiguration.cs [hunk 3]
```diff
@@ -1,17 +1,35 @@
...
-using Microsoft.Bot.Schema;
-using System;
-using System.Collections.Generic;
-using System.Text;
+﻿// Copyright(c) Microsoft Corporation.All rights reserved.
+// Licensed under the MIT License.
 
 namespace Microsoft.BotKit
 {
+    using Microsoft.Bot.Builder;
```
Comment (ceciliaavila) [inline Botkit/Microsoft.BotKit/BotWorkerConfiguration.cs]: The same here with the using Statements.

File: Botkit/Microsoft.BotKit/BotWorker.cs (hunk 2 above)
Comment (Maihav) [reply Botkit/Microsoft.BotKit/BotWorker.cs]: Yes, there is an extra '.' there. But if I remove it, then a warning is shown saying: Documentation text should end with a period. Do you think we can ignore that warning?

File: Botkit/Microsoft.BotKit/BotWorker.cs (hunk 1 above)
Comment (Maihav) [reply Botkit/Microsoft.BotKit/BotWorker.cs]: ACK. I will ignore the rule. Thanks!

File: Botkit/Microsoft.BotKit/BotWorkerConfiguration.cs (hunk 3 above)
Comment (Maihav) [reply Botkit/Microsoft.BotKit/BotWorkerConfiguration.cs]: ACK

File: Botkit/Microsoft.BotKit/BotWorker.cs (hunk 2 above)
Comment (ceciliaavila) [reply Botkit/Microsoft.BotKit/BotWorker.cs]: No, just leave it then. No problem.

File: Botkit/Microsoft.BotKit/BotWorker.cs (hunk 1 above)
Comment (Maihav) [reply Botkit/Microsoft.BotKit/BotWorker.cs]: Done

File: Botkit/Microsoft.BotKit/BotWorkerConfiguration.cs (hunk 3 above)
Comment (Maihav) [reply Botkit/Microsoft.BotKit/BotWorkerConfiguration.cs]: Done

**Pull request merged**